### ISSUE TITLE
Issue with attempting to view manga via the reader.

### ISSUE THREAD

**Issue by Imnewhere12** [tachiyomiorg/tachiyomi-extensions#14978]
### Source information and language

Mangabuddy 1.3.12

### Steps to reproduce

1. Select manga to read.
2. Select chapter to view.
3. Black page with loading circle shows with the page count at the bottom.

### Expected behavior

Page should show images of the manga chapter.

### Actual behavior

Black page with loading circle shows with the page count at the bottom and does not open images it just loads infinitely.

### Tachiyomi version

0.14.3

### Android version

Android 13

### Other details

I've tried multiple steps to resolve or double check the issue.
1. Clear Cache
2. Clear Cookies
3. Delete Webview data 
4. Open the chapter via Webview to confirm everything shows up and isnt blocked by Cloudflare. 
5. Tried to view chapters of manga I read 2 days ago via the extension when this wasnt happening and get the same issue now.
6. Confirmed this is only happening with this Extension not other extensions.

![Screenshot_20230116_104703_Tachiyomi](https://user-images.githubusercontent.com/122816969/212734033-355d7d94-2fb2-45ce-b2d8-0570f36278dc.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by stevenyomi** [comment]
Duplicate of #14967

**Comment by curche** [comment]
Cloudflare auto-response turned out to be false positive

**Comment by curche** [comment]
Duplicate of #14967
